community/cypher/cypher-compiler-2.3/src/main/scala/org/neo4j/cypher/internal/compiler/v2_3/notification/InternalNotification.scala

```diff
@@ -31,3 +31,5 @@ case class CartesianProductNotification(position: InputPosition) extends Interna
 case class LengthOnNonPathNotification(position: InputPosition) extends InternalNotification
 
 case object LegacyPlannerNotification extends InternalNotification
+
+case object CostForUpdatesNotification extends InternalNotification
```

Comment:
CostForUpdates seems confusing to me, had no clue what it would be about when reading the name.